src/components/player/QuizCompleted.tsx

```diff
@@ -0,0 +1,37 @@
+import { Link } from "react-router-dom";
+import { state$ } from "../../state";
+import ScoreEntry from "./ScoreEntry";
+
+function QuizCompleted() {
+  const leaderboard = state$.quiz.leaderboard.get(); // leaderboard of top 3 scores
+  const currentPlayerScore = state$.quiz.playerFinalScore.get(); // current player's score
+  const currentPlayer = state$.player.id.get();
+  const playerPlaced = leaderboard.some((player) => player[0] === currentPlayer); // did player place in top 3?
+  return (
+    <div className="w-full h-full flex flex-col justify-between items-center gap-10">
+      <h1 className="text-6xl">Quiz Completed</h1>
+      <h2 className="text-3xl font-semibold">Leaderboard</h2>
+      <div className="w-96 flex flex-col justify-between items-center gap-2">
+        {leaderboard.map((player, idx) => (
+          <ScoreEntry
+            key={idx}
+            player={player}
+            currentPlayer={currentPlayer}
+            playerScore={player[1]}
+          />
+        ))}
+        {!playerPlaced && (
+          <div className="w-full flex justify-between text-2xl p-4 rounded-xl bg-yellow-600">
```

Comment:
You should be using the `ScoreEntry` component, so I suggested making the component. You do not have to keep repeating yourself.